scripts/templates/csv-template.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: dedicated-admin-operator-0.0.1
```

Comment:
For properties we're overriding I'd rather the value in this template be something we know will be replaced.  For example, name will become something like `dedicated-admin-operator.v0.1.29-22b5144` so in the template can we say `dedicated-admin-operator-v$FULL_VERSION`?  Other fields that are replaced I would change too.

The why is in case there's a regression it's obvious the value wasn't updated correctly.

Reply:
Updated the template file to make it explicit the fields we'll be replacing